tests/ref.txt

```diff
@@ -0,0 +1,124 @@
+    "cmip6:source_id": "CanESM5",
+    "cmip6:source_type": [
+      "AOGCM"
+    ],
+    "cmip6:sub_experiment": "none",
+    "cmip6:sub_experiment_id": "none",
+    "cmip6:table_id": "SImon",
+    "cmip6:variable_id": "siconc",
+    "cmip6:variant_label": "r13i1p2f1",
+    "cmip6:initialization_index": 1,
+    "cmip6:physics_index": 2,
+    "cmip6:realization_index": 13,
+    "cmip6:forcing_index": 1,
+    "cmip6:tracking_id": "hdl:21.14100/9e4f804b-c161-44fa-acd1-c2e94e220c95",
+    "cmip6:version": "v20190429",
+    "cmip6:product": "model-output",
+    "cmip6:license": "CMIP6 model data produced by The Government of Canada (Canadian Centre for Climate Modelling and Analysis, Environment and Climate Change Canada) is licensed under a Creative Commons Attribution ShareAlike 4.0 International License (https://creativecommons.org/licenses). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing CMIP6 output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file) and at https:///pcmdi.llnl.gov/. The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law.",
+    "cmip6:grid": "ORCA1 tripolar grid, 1 deg with refinement to 1/3 deg within 20 degrees of the equator; 361 x 290 longitude/latitude; 45 vertical levels; top grid cell 0-6.19 m",
+    "cmip6:mip_era": "CMIP6"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          0.049800001084804535,
+          -78.39350128173828
+        ],
+        [
+          0.049800001084804535,
+          89.74176788330078
+        ],
+        [
+          359.99493408203125,
+          89.74176788330078
+        ],
+        [
+          359.99493408203125,
+          -78.39350128173828
+        ],
+        [
+          0.049800001084804535,
+          -78.39350128173828
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "source",
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/fileServer/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc",
+      "type": "application/x-netcdf",
+      "title": "birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
```

Comment:
Still missing the `thredds:` prefix.

Reply:
Just to be clear, here `thredds:` refers to the specific service name from `https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/`. If another THREDDS service was hosted as `https://pavics.ouranos.ca/twitcher/ows/proxy/custom-thredds/`, the title would have `custom-thredds:path/to/file.nc` instead.